### PR TITLE
Editing the HomePage at Feras_MO site

### PR DESCRIPTION
**Pull request by Albadro**
# Editing the HomePage at Feras_MO site

### here are the done changes (commits):

-   Delete unused files and adjust files names and pathes.
-   Enhance index.html, changes are:
    -   remove the useless elemnts.
    -   remove inline styles.
    -   add needed elements and attributes for styling.
-   Enhance style.css, changes are:
    -   auto format(vscode auto format).
    -   remove some of useless styles.
    -   adjust paddings and margins.
    -   centering and adjusting the movement of the moving channel_profile photo.
    -   resizing the gear btn.
    -   adding a border radius to buttons.
    -   add duration to buttons hover effect.
-   U r the dumb one here

> Mr.Feras,  
> thanks for surfing my commits changes and reading the readme.md, I hope you like it.

### CODE REVIEW

**Comment by Albadro** [comment]
don't forget to edit the readme.md after merging to make it suit the whole project, don't text me on whatsapp about this ever, act like if you are a stranger and I'm making pull request to his repo, talk about it here only, thanks.

**Review by Feras-MO — COMMENTED** [review]
first i don't see any improvement in the project all i see is tags doesnt work or u forget to close them so try fix ur edits

**Comment by Albadro** [comment]
Have you read the pull request discription?! or even looked around in the codes!
- if you do you'll find out what the enhancements are, you can see it visually on [github page of my fork](https://albadro.github.io/feras-channel/).

- Also, you can easily edit the range of the movement of the moving prof_pic (cuz it may look very fast on wider screens) by editing the margin of the new element I've added (**.movemey-container**),on media queries.

**Comment by Albadro** [comment]
and wait, what do you mean with the tags that doesn't work?? ...
I didn't get to the point

**Comment by Albadro** [comment]
U can install github on Android for faster communication (it is only **12MB**)

**Comment by Albadro** [comment]
# Hey!!!!!!!!!!!, U alive?

File: css/style.css
Comment (Albadro) [inline css/style.css:1]: you can see there are much of the added and deleted lines because of the vs code auto format

**Pull request merged**